src/redux/reducers/users.js

```diff
@@ -22,37 +23,27 @@ export const usersReducer = (state = initialState, action) => {
         ...state,
         users: action.payload
       };
-    case FOLLOW:
+    case FOLLOW_USERS_USER:
       return {
         ...state,
-        users: state.users.map(user => {
-          if (user.id === action.payload) {
-            return {...user, followed: true}
-          }
-          return user;
-        })
+        users: updateObjectInArray(state.users, action.payload, {followed: true})
       };
-    case UNFOLLOW:
+    case UNFOLLOW_USERS_USER:
```

Comment:
CAn we name it just UNFOLLOW_USER? =)

Reply:
USERS is the name of the reducer.
UNFOLLOW_USER are too common names and can be used for other reducers.
We need to name the action type more unique way.
I am use the schema [action]_[reducer]_[subject] here.